lualib/ngx/ssl/session/ticket/key_rotation.lua

```diff
@@ -57,52 +57,15 @@ end
 
 local shdict_name, shm_cache_pos_ttl, shm_cache_neg_ttl, disable_shm_cache
```

Comment:
These variables seems no need stay in the top level, maybe we can just move them to `init` function?